src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -30,6 +30,12 @@
 __all__ = ["TextGenerationPipeline"]
 
 
+PROMPT_PREFILL = "engine_prompt_prefill"
```

Comment:
I do not like those variables "floating around" in the script. This Python file is already fairly complex and does not need more clutter. Could we possibly introduce more order to the definitions of the timing stage names? 
A simple idea would be to wrap those four variables inside a data class (like here https://github.com/neuralmagic/deepsparse/blob/main/src/deepsparse/utils/timer.py#L29)  and import from a helper script?